wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -269,6 +269,18 @@ pub enum WalletCommand {
         label: Option<String>,
     },
 
+    #[clap(name = "account-add-standalone-multisig")]
```

Comment:
Two things:

1. Please change all functions that have to do with standalone to have a prefix `standalone-`, not `account-`.
2. Please rename this function `standalone-create-multisig`